administrator/components/com_associations/models/forms/filter_associations.xml

```diff
@@ -52,6 +51,8 @@
 				>
 				JASSOCIATIONS_DESC
 			</option>
+			<option value="a.menutype ASC">COM_MENUS_HEADING_MENU_ASC</option>
+			<option value="a.menutype DESC">COM_MENUS_HEADING_MENU_DESC</option>
```

Comment:
hum maybe we need something like `requires` for disabling options is the list options...
This menu ordering should only be available when menu component is selected.